gef.py

```diff
@@ -4081,16 +4060,16 @@ def __init__(self, func: str, retval: Optional[int]) -> None:
         self.func = func
         self.retval = retval
 
-        m = "All calls to '{:s}' will be skipped".format(self.func)
+        m = f"All calls to '{self.func}' will be skipped"
         if self.retval is not None:
-            m += " (with return value set to {:#x})".format(self.retval)
+            m += f" (with return value set to {self.retval:#x})"
         info(m)
         return
 
     def stop(self) -> bool:
-        m = "Ignoring call to '{:s}' ".format(self.func)
-        m += "(setting return value to {:#x})".format(self.retval)
-        gdb.execute("return (unsigned int){:#x}".format(self.retval))
+        m = f"Ignoring call to '{self.func}' "
+        m += f"(setting return value to {self.retval:#x})"
```

Comment:
we can instead do
```python
m = (f"..."
     f"...")
```

Reply:
will do! in this case the whole variable should be inlined as well IMO